processor/processor.go

```diff
@@ -1088,24 +1088,59 @@ func (proc *Handle) updateMetricMaps(
 	sd.ViolationCount += int64(veCount)
 }
 
-func (proc *Handle) getNonSuccessfulMetrics(response transformer.Response, commonMetaData *transformer.Metadata, eventsByMessageID map[string]types.SingularEventWithReceivedAt, inPU, pu string) *NonSuccessfulTransformationMetrics {
+func (proc *Handle) getNonSuccessfulMetrics(
+	response transformer.Response,
+	commonMetaData *transformer.Metadata,
+	eventsByMessageID map[string]types.SingularEventWithReceivedAt,
+	inPU, pu string,
+) *NonSuccessfulTransformationMetrics {
```

Comment:
**Personal preference:** I don't find that by splitting a single line, that fits perfectly in the screen, into multiple lines makes the codebase more readable

Reply:
It's not fitting on my screen, I'm having to turn my head and sometimes worse: scroll!
To each their own though, can revert if it's becoming a nuisance for everyone